### PR TITLE
adding new unit name mappings for OESS for QinQ signaled circuits

### PR DESCRIPTION
**Pull request by ajragusa**
This should resolve both #595 and #593

### CODE REVIEW

File: perl-lib/OESS/lib/OESS/DB/VRF.pm [hunk 1]
```diff
@@ -145,20 +145,64 @@ sub delete_endpoints {
     return $ok;
 } 
 
+sub find_available_unit{
+    my %params = @_;
+    my $db = $params{'db'};
+    my $interface_id = $params{'interface_id'};
+    my $tag = $params{'tag'};
```
Comment (jonstout) [inline perl-lib/OESS/lib/OESS/DB/VRF.pm]: tag and inner_tag aren't used in this method. they can be removed

File: perl-lib/OESS/lib/OESS/Database.pm [hunk 2]
```diff
@@ -213,6 +213,45 @@ sub reconnect{
 
 }
 
+
+sub find_available_unit{
+    my $self = shift;
+    my %params = @_;
+    my $interface_id = $params{'interface_id'};
+    my $tag = $params{'tag'};
```
Comment (jonstout) [inline perl-lib/OESS/lib/OESS/Database.pm]: tag and inner_tag aren't used in this method. they can be removed

File: perl-lib/OESS/share/nddi.sql [hunk 3]
```diff
@@ -73,6 +73,7 @@ CREATE TABLE `circuit_edge_interface_membership` (
   `extern_vlan_id` int(10) NOT NULL,
   `inner_tag` int(10) DEFAULT NULL,
   `circuit_edge_id` int(10) NOT NULL AUTO_INCREMENT,
+  `unit_name` int(11) NOT NULL,
```
Comment (jonstout) [inline perl-lib/OESS/share/nddi.sql]: You're using `select unit from circuit_edge_interface_membership` in the `find_available_unit` methods. Column names must be consistent.

File: perl-lib/OESS/lib/OESS/DB/VRF.pm (hunk 1 above)
Comment (ajragusa) [reply perl-lib/OESS/lib/OESS/DB/VRF.pm]: I left these for future use (if we need them later)

**Pull request merged**